package.json

```diff
@@ -12,12 +12,25 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@src.temando.io:developer-experience/temando-open-api-ui.git"
+    "url": "git@github.com:temando/open-api-ui.git"
   },
   "keywords": [
     "ui"
   ],
-  "author": "Florie Guibert",
+  "contributors": [
+    {
+      "name": "Florie Guibert",
+      "email": "florie.guibert@temando.com"
+    },
+    {
+      "name": "Brendan Abbot",
+      "email": "brendan.abbott@temando.com"
+    },
+    {
+      "name": "Khoa Tran",
+      "email": "khoa.tran@temando.com"
+    }
+  ],
   "license": "ISC",
```

Comment:
LICENCE is MIT, but this states ISC. I'm not fussed either way, but just needs to be consistent